app/src/ui/updates/update-available.tsx

```diff
@@ -25,7 +25,7 @@ export class UpdateAvailable extends React.Component<
           An updated version of GitHub Desktop is available and will be
           installed at the next launch. See{' '}
           <LinkButton uri={this.props.releaseNotesLink}>what's new</LinkButton>{' '}
-          or <LinkButton onClick={this.updateNow}>restart now</LinkButton>.
+          or <LinkButton onClick={this.updateNow}>restart GitHub now</LinkButton>.
```

Comment:
If we're going to be more explicit here to avoid confusion, I'd rather this be updated to `GitHub Desktop` to match with the installed program name - GitHub could be confused with the website or the company.

Reply:
That's great, but I have an amateur question. All that was added was a word, so how come all the build checks failed?

Reply:
@AB1908 we use [Prettier](https://github.com/prettier/prettier) to ensure our code is formatted in a consistent way, and the additional text now means that Prettier wants it on the next line:

```diff
diff --git a/app/src/ui/updates/update-available.tsx b/app/src/ui/updates/update-available.tsx
index dd2c711a9..6f2fd7813 100644
--- a/app/src/ui/updates/update-available.tsx
+++ b/app/src/ui/updates/update-available.tsx
@@ -25,7 +25,8 @@ export class UpdateAvailable extends React.Component<
           An updated version of GitHub Desktop is available and will be
           installed at the next launch. See{' '}
           <LinkButton uri={this.props.releaseNotesLink}>what's new</LinkButton>{' '}
-          or <LinkButton onClick={this.updateNow}>restart GitHub now</LinkButton>.
+          or{' '}
+          <LinkButton onClick={this.updateNow}>restart GitHub now</LinkButton>.
         </span>
 
         <a className="close" onClick={this.dismiss}>
```

If you run `yarn lint:prettier --fix` that should apply the change to your branch.

Reply:
Ah that makes sense. Thanks for explaining.